src/cbc_sdk/platform/users.py

```diff
@@ -149,13 +162,8 @@ def add_grant_profile(self, orgs, roles):
             return self
 
         def build(self):
-            """
-            Builds the new user and returns it.
-
-            Returns:
-                User: The new user object from the server.
-            """
-            return User._create_user(self._cb, self._creation_data)
+            """Builds the new user."""
+            User._create_user(self._cb, self._creation_data)
```

Comment:
You added a comment below the query implementation I would add something similar to the docstring for context 

```
# N.B.: new user is not "findable" until activated and initial password set
```

Reply:
I fleshed the docstring out a bit along these lines.

Reply:
Can we add the same note to the `def build` function as well

Reply:
Copy-pasta done.